docker-compose.yml

```diff
@@ -0,0 +1,12 @@
+version: '3.7'
+
+services:
+    app:
```

Comment:
What naming conditions would you suggest?
I'm naming it `app` after its role, so if I used a Postgres image its name would be `db`.

Reply:
I renamed it to `time_tracking`

I usually name the container as I explained earlier because as you know, docker-compose abbreviate the names of the containers with the folder name, so on my computer, the container name is `time-tracking_app`.